### ISSUE TITLE
Bug: Flyout doesn't appear when using gesture audio controls

### ISSUE THREAD

**Issue by HenriHawk42** [ModernFlyouts-Community/ModernFlyouts#1190]
I use touchpad gestures to control my volume and media on windows 11, and I've noticed an issue in which modernflyouts does not appear when using those gestures, it only works when using the media keys.

Perhaps this is caused by modernflyouts listening for media key presses, rather than a change to the system media state? I am not really sure how this program works.

**Comment by LeoMacherla** [comment]
Yes I'm experiencing this too. I thought that it wasn't showing the flyout at all, but I realise that you're right. It is only when using touchpad gestures (or not using the media keys)
